### ISSUE TITLE
excessive traffic loss

### ISSUE THREAD

**Issue by Emersonn022** [dvygolov/YellowCloaker#44]
Hello, I'm advertising on tiktok, and I added cloaker to my structure, but I'm experiencing a lot of traffic loss. 
Real example: I have 100 target clicks in my campaign, these clicks are listed in the allowed part within the cloaker, but on my black page they only reach around 15 to 20 of them. Is this a cloaker bug? Could you help me? Thank you, have a nice day.

**Comment by dvygolov** [comment]
Hello, is your black page on some other domain and you redirect to it from
the cloaker? If the clicks are listed in the Allowed then it means that
they reached the server and are not lost. How do you know that only 15-20
clicks were shown the black page? Do you have some other methods to measure
clicks on your black page?

**Comment by Emersonn022** [comment]
Yes, my Black page ( my shopify store ) is on a domain and I am using cloaker on another domain other than the one where my black page is, but in cloaker I am redirecting to that domain where my black page is. Should I use cloaker on the same domain as my black page? Could this be the problem with the loss of traffic? Regarding measuring clicks, I use Shopify so I see it through their dashboard.

**Comment by Emersonn022** [comment]
Yes, my Black page ( my shopify store ) is on a domain and I am using cloaker on another domain other than the one where my black page is, but in cloaker I am redirecting to that domain where my black page is. Should I use cloaker on the same domain as my black page? Could this be the problem with the loss of traffic? Regarding measuring clicks, I use Shopify so I see it through their dashboard.
________________________________
De: Daniel Vygolov ***@***.***>
Enviado: sexta-feira, 16 de agosto de 2024 07:15
Para: dvygolov/YellowCloaker ***@***.***>
Cc: Emersonn022 ***@***.***>; Author ***@***.***>
Assunto: Re: [dvygolov/YellowCloaker] excessive traffic loss (Issue #44)


Hello, is your black page on some other domain and you redirect to it from
the cloaker? If the clicks are listed in the Allowed then it means that
they reached the server and are not lost. How do you know that only 15-20
clicks were shown the black page? Do you have some other methods to measure
clicks on your black page?

**Comment by dvygolov** [comment]
You should measure the speed of your redirects using any browser extension,
for example Link Redirect Trace. You'll see how long does it take to load
your shopify. I think that the click loss can be caused by slow redirect
speed.
